core/templates/components/skill-selector/skill-selector.directive.html

```diff
@@ -62,12 +62,42 @@ <h3><label class="list-group-item-heading" style="padding-left:10px"><[topicName
   </div>
 </div>
 <style>
-  .skill-selector-parent .skill-list-box {
+  select-skill .skill-selector-parent
+  select-skill .skill-list-box {
     max-height: 50vh;
     overflow-y: auto;
   }
+  select-skill .oppia-skill-container {
+    display: flex;
+    flex-direction: row;
+  }
+  select-skill .oppia-list-group {
+    width: 70%;
+  }
+  select-skill .oppia-topic-name-text
+  select-skill .oppia-heading-text
```

Comment:
Are these working? These need to be seperated by commas,
```
  select-skill .oppia-topic-name-text,
  select-skill .oppia-heading-text,
  select-skill .oppia-list-view-item {
```

Reply:
Done.